src/main/scala/com/lightbend/conductr/sbt/ConductRImport.scala

```diff
@@ -34,6 +34,10 @@ object ConductrImport {
   )
 
   object ConductrKeys {
+    val dryRun = SettingKey[Boolean](
```

Comment:
Two of the tests set this to `true` because they are only intended to test the argument parsing, not the task itself, so this mechanism basically disables the task execution and it just returns.